pythonx/jedi_vim.py

```diff
@@ -148,6 +157,36 @@ def wrapper(*args, **kwargs):
     return func_receiver
 
 
+last_force_python_error = None
+
+
+def get_environment():
+    global last_force_python_error
+
+    force_python_version = vim_eval("g:jedi#force_py_version")
```

Comment:
This could optionally become a buffer-local variable, so that you could use different versions per buffer then.
`vim_eval("get(b:, 'jedi_force_py_version', get(g:, 'jedi#force_py_version')")`

Reply:
I agree. I just think that this should be part of another pull request.